webapp/tabula_web.rb

```diff
@@ -92,6 +92,155 @@ def upload(file)
   return [job_batch, file_id]
 end
 
+def list_templates
```

Comment:
I guess the integration would be easier if you make a new PR against tabula-java-1.0.0 instead of master.